draft-ietf-ppm-dap.md

```diff
@@ -2318,11 +2240,10 @@ determined by the query type. If the size check fails, then Helpers MUST abort
 with an error of type "invalidBatchSize". Leaders SHOULD wait for more reports
 to be validated and try the collection job again later.
 
-Next, the Aggregator checks that the batch has not been queried too many times.
-This is determined by the maximum number of times a batch can be queried,
-`max_batch_query_count`. If the batch has been queried with more than
-`max_batch_query_count` distinct aggregation parameters, the Aggregator MUST
-abort with error of type "batchQueriedTooManyTimes".
+Next, the Aggregator checks that the batch has not been queried with multiple
+distinct aggregation parameters. If the batch has been queried with more than
+one distinct aggregation parameter, the Aggregator MUST abort with error of type
+"batchQueriedMultipleTimes".
```

Comment:
I think this could be stated more clearly in terms of the batch already having been collected.
```suggestion
Next, the Aggregator checks that the batch has not already been aggregated. If it
has, then the Aggregator MUST abort with error of type "batchAlreadyAggregated".
```

Note also the error type changes `batchQueriedMultipleTimes => batchAlreadyAggregated`.